index.html

```diff
@@ -233,6 +233,59 @@
       </p>
       <section>
         <h3>
+          Installing via automated prompting
+        </h3>
+        <p>
+          User agents that support an <a>automated install prompt</a> MUST run
+          the following steps before initiating an install process:
+        </p>
+        <ol>
+          <li>If a <a>manually</a> instantiated <a>installation process</a> for
+          this web application, terminate this algorithm.
+          </li>
+          <li>Let <var>event</var> be a newly constructed
+          <a>BeforeInstallPromptEvent</a> event named "beforeinstallprompt",
+          which does not bubble and is cancellable.
+          </li>
+          <li>Let <var>prompt promise</var> be the promise corresponding to the
+          event's prompt() method.
+          </li>
+          <li>Let <var>user-choice promise</var> be the promise corresponding
+          to the event's <code>prompt()</code> method.
+          </li>
+          <li>Fire <var>event</var> at the <code>Window</code> object.
+          </li>
+          <li>Queue a task on the Realm’s settings object’s responsible event
+          loop:
+            <ol>
+              <li>If <var>event</var>'s <a>cancel flag</a> is set, terminate
+              this algorithm.
+              </li>
+              <li>Otherwise, attempt to present the end-user with a UI that
+              lets them choose if they would like to install the web
+              application.
+              </li>
+              <li>If the previous step succeeds, resolve <var>prompt
+              promise</var>. Otherwise, reject <var>prompt promise</var> with
+              an ""AbortError", or "OperationError", or "NotAllowedError", or
```

Comment:
Yep, will add all the links... don't want to add them yet as will have to rewrite this a few times yet.